docs/site/Model.md

```diff
@@ -473,6 +473,8 @@ class Product extends Entity {
 }
 ```
 
+> NOTE: Currently, property types must be specified explicitly either on the property itself or via the `type` option of the property decorator. Aliased types or types that extracted from a class or interface (e.g. `public name: OtherClass["otherProperty"]`) will not work properly and will result in the property type being resolved as an empty object rather than the intended type in the generated OpenAPI specifcation. This is due to a limitation and flaw in the way TypeScript currently generates the metadata that is used to generate the OpenAPI specification for the application. (see [Issue #3863](https://github.com/strongloop/loopback-next/issues/3863))
```

Comment:
Have you considered using a "Note alert" as described in our contrib docs [Alerts](https://loopback.io/doc/en/contrib/alerts.html)?

I think it will render more nicely.

Reply:
We usually go with the following style:

```
{% include note.html content="
This is my note. All the content I type here is treated as a single paragraph.
" %}
```